src/main/java/com/epam/pages/popup/SubjectPopup.java

```diff
@@ -1,6 +1,127 @@
 package com.epam.pages.popup;
 
+import com.epam.helpers.SharedTestData;
 import com.epam.pages.common.CommonPopup;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.List;
 
 public class SubjectPopup extends CommonPopup {
+
+    @FindBy(id = "name")
+    private WebElement nameField;
+    @FindBy(id = "teacher")
+    private WebElement teacherSelect;
+    @FindBy(xpath = "//*[@class='selection']/span")
+    private WebElement selectedTeachers;
```

Comment:
I think we should change the name of selectedTeachers WebElement, because element with the locator shows select teacher BOX and not selected teachers in it... I suggest to rename it to selectTeachersBox

Reply:
Resolved